scipy/stats/_distribution_infrastructure.py

```diff
@@ -3226,7 +3346,7 @@ def logintegrand(x, order, logcenter, **params):
 
     ### Convenience
 
-    def plot(self, x='x', y='pdf', *, t=('cdf', 0.0005, 0.9995), ax=None):
+    def plot(self, x='x', y=None, *, t=None, ax=None):
```

Comment:
Lots of changes here to make pretty plots of discrete distribution PMFs / CDFs. This also improves the handling of distributions with finite support endpoints.